### PR TITLE
Sketch of post re: talk marketing

### PR DESCRIPTION
**Pull request by jennybc**
As previously discussed. It needs more content, examples, and ... good humor? I think I sound like someone already got on my last nerve 😬 but I wanted to get this rolling so we could post early next week, if it's a go.

### CODE REVIEW

**Comment by jennybc** [comment]
Some self-critique: needs to be clear that this is about narrow and specific details re: talk prep. It presumes that the thing is already at the level desired level of quality and that you've already created good documentation.

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Feature slides
+
+Admit that people photograph and share individual slides, e.g., via Twitter. This can be a good thing, especially if you harness it for your goals! If an audience member is moved to share your thing, it is natural they want to enhance the basic message with an evocative slide or image. Think about how well your slides work with this well-established behaviour.
+
+If something would be terrible taken out of context or circulated widely, reflect on that and take appropriate measures.
+
+If you have a pithy and hilarious point, give that slide some extra attention.
+
+*Link to some guides on live-tweeting? If you'd be chuffed to see people tweet about your talk, view your talk through their eyes. This is like grant-writing: notice the words the funder uses to describe what they want to fund and repeat them back, verbatim. Find out what a live-tweeter needs and give them exactly that.*
```
Comment (batpigandme) [inline content/articles/2018-07-talk-carpe-diem.markdown]: Emily Robinson has tips on live tweeting in this post: http://hookedondata.org/Building-Your-Data-Science-Network-Finding-Community/ (possible that Dave Robinson actually has slides on live tweeting, I'll ask)

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Post it before the talk
+
+Choose your primary online home and secure a shortlink at it BEFORE YOUR TALK. Do not promise the slides or materials "after the talk". People are taking notes and photos right now, during your talk, and this is your best chance to tell them how to find your awesome thing when they get home.
+
+Even if you add more materials or update your slides after the talk, it is still worth it to publicize the shortlink during the talk. These tweets will live on for months and wrap-up blog posts will come out over the next couple of weeks. Most of the genuine visits to your thing will happen later, but you will never have such a comparable opportunity to put the URL in front of people's eyeballs.
```
Comment (batpigandme) [inline content/articles/2018-07-talk-carpe-diem.markdown]: "you will never have a comparable opportunity"

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Share slides on a slide-sharing platform
+
+Consider how easy it is for a wide audience to click through your slides with minimum friction. Slide-specific platforms, such as [SpeakerDeck](https://speakerdeck.com), are ideal for quick browsing and linking to specific slides.
```
Comment (batpigandme) [inline content/articles/2018-07-talk-carpe-diem.markdown]: (These can also be easily embedded in a blog post with a blogdown shortcode).

File: content/articles/2018-07-talk-carpe-diem.markdown (hunk 1 above)
Comment (batpigandme) [reply content/articles/2018-07-talk-carpe-diem.markdown]: Or just the tweet of drob's slide on live-tweeting, I guess: https://twitter.com/zabormetrics/status/855518845779496961?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E855518845779496961&ref_url=http%3A%2F%2Fvarianceexplained.org%2Fr%2Fnyr-conference%2F

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+*We assume your product meets your quality standards and that you have already documented it. This is about stuff you can do in ~1 hour as you prepare you talk that will have high payoff in terms of recruiting users. This is also not about how to give a great talk.*
+
+## Marketing is not a Four Letter Word
+
+First, let's deal with the word and concept of "marketing".
+
+> Marketing is the study and management of exchange relationships.
+
+https://en.wikipedia.org/wiki/Marketing
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: I think we usually put an em-dash before quote sources

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+If you have taken the trouble to build an R package, Shiny app, tutorial, or talk, presumably you would like people to know about it and use it! Marketing is the set of activities that help your target audience
+
+  * discover that your thing exists
+  * learn how your thing could benefit them
```
[the remark this answers is not in the document]
Comment (hadley) [reply content/articles/2018-07-talk-carpe-diem.markdown]: In that case, also needs a comma after "exists"

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## Where does your thing live?
+
+Determine the primary online home for your product. Possibilities:
+
+  * A custom website, possibly made with [pkgdown](http://pkgdown.r-lib.org), [blogdown](https://bookdown.org/yihui/blogdown/), [bookdown](https://bookdown.org/yihui/bookdown/), or, in the case of [Shiny](https://shiny.rstudio.com), a [running instance of the app itself](https://www.shinyapps.io)
+  * A development venue, such as a [GitHub repository](https://github.com)
+  * A [CRAN](https://cran.r-project.org/web/packages/index.html) package landing page
+
+Visit this home with the eyes of a newcomer and refine accordingly. If your thing has multiple online homes, make sure they link to each other, with a strong current towards the primary home.
```
[the remark this answers is not in the document]
Comment (hadley) [reply content/articles/2018-07-talk-carpe-diem.markdown]: current -> push maybe? I read as electrical current initially.

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+## Make a shortlink
+
+Unless your primary URL is already extremely short and memorable, consider creating a shortlink that is both. A link in a talk will be captured in cell phone photos and typed with fingers into phones. Make it easy on people!
+
+http://bit.ly
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: Needs lead into to link?

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Title and closing slide
+
+Make a slide with the most important details on you and your thing. Include this very early in your talk (probably first?) and repeat some variant at the end. This slide should feature your shortlink and any relevant URLs or handles, e.g. Twitter, GitHub, website, or email address.
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: This slide should feature:

* your name
* how to contact you (e.g. twitter handle, email)
* name of your thing
* how to find your thing (e.g. website, GitHub repo)

If you can squeeze in a quick description of your thing, or a visual summary of your talk, do so! That way people will get a little context for free when they come back to the photo.

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+DO THIS IN AN ABSOLUTELY HUGE FONT. Now is not the time for subtlety or mumbling "I know it's hard to read, but ...".
+
+Leave this slide up for a bigly amount of time, so that people have ample opportunity to get out their phones and fire up the camera. Do not flash this vital slide for 3 seconds, only to spend 2 minutes displaying a bulleted outline of your talk.
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: This sort of slide is great to leave up during the introduction to your talk, and while you take questions.

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+## Feature slides
+
+Admit that people photograph and share individual slides, e.g., via Twitter. This can be a good thing, especially if you harness it for your goals! If an audience member is moved to share your thing, it is natural they want to enhance the basic message with an evocative slide or image. Think about how well your slides work with this well-established behaviour.
+
+If something would be terrible taken out of context or circulated widely, reflect on that and take appropriate measures.
+
+If you have a pithy and hilarious point, give that slide some extra attention.
+
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: One thing that you've done which I really like is to include a link to the presentation in the footer of every slide

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+Even if you add more materials or update your slides after the talk, it is still worth it to publicize the shortlink during the talk. These tweets will live on for months and wrap-up blog posts will come out over the next couple of weeks. Most of the genuine visits to your thing will happen later, but you will never have such a comparable opportunity to put the URL in front of people's eyeballs.
+
+## Share slides on a slide-sharing platform
+
+Consider how easy it is for a wide audience to click through your slides with minimum friction. Slide-specific platforms, such as [SpeakerDeck](https://speakerdeck.com), are ideal for quick browsing and linking to specific slides.
+
+The way you interact with and present your slides is not necessarily a great distribution platform. Large PDFs in GitHub repo do not offer a pleasant browsing experience and raw HTML slides on GitHub are even worse, i.e. are not viewable.
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: In short, think about how the people who you want to use your stuff, and make their lives easier!

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 12]
```diff
@@ -0,0 +1,82 @@
+---
+title: Talk Carpe Diem
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: Carpe Talk ?

File: content/articles/2018-07-talk-carpe-diem.markdown [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+If you have taken the trouble to build an R package, Shiny app, tutorial, or talk, presumably you would like people to know about it and use it! Marketing is the set of activities that help your target audience
+
+  * discover that your thing exists
+  * learn how your thing could benefit them
+
+Use your talk (and documentation and README, etc.) to accomplish these goals!
+
+Some creators take a peculiar pride in being "bad at marketing", but it is not some elusive skill. Nor is marketing ability anti-correlated with creating a high-quality product. You can acknowledge the value in marketing or you can declare that cultivating broader usage is not your goal. Pick one.
+
```
Comment (hadley) [inline content/articles/2018-07-talk-carpe-diem.markdown]: Point to some general marketing advice? I like https://medium.com/indeed-data-science/marketing-for-data-science-a-7-step-go-to-market-plan-for-your-next-data-product-60c034c34d55 (written by Roger's son)

File: content/articles/2018-07-talk-carpe-diem.markdown (hunk 8 above)
Comment (batpigandme) [reply content/articles/2018-07-talk-carpe-diem.markdown]: Not here, per se, but I think we might want to add something about visuals generally. Two examples that come to mind are @riannone's gt slides (though those aren't public, I guess), and Garrick Aden-Buie's ggplot2 deck (specifically the progression from 21-26 as it intermingles code and visuals nicely https://pkg.garrickadenbuie.com/trug-ggplot2/#21 — also I think they're reveal.js or xarigan, so it's not a matter of having fancy graphics programs or dealing with keynote).

**Comment by jennybc** [comment]
@hadley @batpigandme I've done a lot more editing. At this point, my only remaining ambition is to take a couple screenshots and drop them in. I started to summarize all the GitHub suggestions but it was just too much. I link to it and will mine it for a couple visuals.

I'd appreciate and act on more feedback from you, but can also just take it from here. I think I should publish before I turn in tonight (Sunday night in Australia), so it's up by Monday morning in Australia.

**Comment by jennybc** [comment]
I am too sleepy to add screenshots. Will publish now, but I note we could still add them later.

**Pull request merged**